enterprise-suite/console_test/testenv/testenv.go

```diff
@@ -0,0 +1,75 @@
+package testenv
```

Comment:
I don't think we need a separate package for this one file. What about sticking it in the `tests` package?

Reply:
If we keep `args` in a separate package, this new `tests` package would only have the current contents of `testenv`, so maybe better to keep the less general name?